cardano-config/src/Cardano/TracingOrphanInstances/Network.hs

```diff
@@ -507,8 +507,10 @@ instance Show peer
       => ToObject [TraceLabelPeer peer (FetchDecision [Point header])] where
   toObject MinimalVerbosity _ = emptyObject
   toObject _ [] = emptyObject
-  toObject _ (lbl : r) = toObject MaximalVerbosity lbl <>
```

Comment:
This `<>` was smashing the entire list of objects into a single object, overriding the Object's map on each iteration..